boost-gradle/src/test/resources/test-jpa/src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<persistence version="2.0"
+             xmlns="http://java.sun.com/xml/ns/persistence"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd">
+
+  <persistence-unit name="jpasamplepersistenceunit">
+
+    <jta-data-source>java:comp/env/jdbc/samplejpadatasource</jta-data-source>
+    <non-jta-data-source>java:comp/env/jdbc/samplejpadatasourcenonjta</non-jta-data-source>
+
+    <class>application.servlet.Thing</class>
+    <exclude-unlisted-classes>true</exclude-unlisted-classes>
+    <properties>
+      <!-- These properties are creating the database on the fly. We are using them to avoid users having
+          to create a database to run the sample. 
+          See also the create=true line in the datasource meta data. -->
+      <property name="openjpa.jdbc.SynchronizeMappings" value="buildSchema(ForeignKeys=true)" />
+      <property name="openjpa.jdbc.DBDictionary" value="derby" />
```

Comment:
My JPA knowledge is pretty slim, but how does this relate to the JDBC booster also creating a DB?  Or is that DB not used?

Reply:
That's a good point. I took this `persistence.xml` from one of our sample projects on wasdev and I didn't notice this. Based on my experimentation it looks like the application is using the JDBC driver (it fails without the driver config) but possibly not the default database we are configuring. I'll look into this more.

Reply:
This is fixed in https://github.com/OpenLiberty/boost/pull/270/commits/9d03eaec65bfd917d8a78ecc48fdcc0051a2c43f.